server.py

```diff
@@ -232,17 +340,24 @@ def contest_ranking(code):
         .switch()
         .join(ContestProblems, on=(Submission.contestProblem == ContestProblems.id))
         .group_by(Submission.user)
-        .order_by(fn.count(Submission.contestProblem.distinct()).desc())
+        .order_by(fn.count(Submission.contestProblem.distinct()).desc())# descending order on basis of no of solution
     )
     order = list(order.tuples())
     order = [
         (username, score, rank) for rank, (username, score) in enumerate(order, start=1)
-    ]
+    ]# associate  ranks to each user
     return bottle.template("rankings.html", people=order)
 
 
+
 @app.get("/ranking")
 def rankings():
+    """
```

Comment:
a better thing to do would have been to rename the function something like `user_rank` since the last one was `contest_rank`

Reply:
right now this docstring is noise